api/net/Areas/Subscriber/Controllers/ContentController.cs

```diff
@@ -147,24 +147,19 @@ public IActionResult DownloadFile(long id)
     /// <param name="path"></param>
     /// <returns></returns>
     [HttpGet("stream")]
-    [ProducesResponseType(typeof(OkObjectResult), (int)HttpStatusCode.OK)]
-    [ProducesResponseType(typeof(OkObjectResult), (int)HttpStatusCode.PartialContent)]
+    [ProducesResponseType(typeof(FileStreamResult), (int)HttpStatusCode.OK)]
+    [ProducesResponseType(typeof(FileStreamResult), (int)HttpStatusCode.PartialContent)]
     [ProducesResponseType((int)HttpStatusCode.BadRequest)]
     [SwaggerOperation(Tags = new[] { "Content" })]
-    public async Task<IActionResult> StreamAsync([FromQuery] string path)
+    public IActionResult StreamAsync([FromQuery] string path)
```

Comment:
This is no longer an async function, rename to `Stream`

Reply:
Updated, thanks